### PR TITLE
Add annotations without indentation

### PR DESCRIPTION
**Pull request by nimakarimipour**
This PR updates injector to add annotations without indentation considerations since it adds unnecessary complications. All indentations can fixed by the using formatter in the target project in the end. It will also simplify #103.

### CODE REVIEW

File: injector/src/test/java/edu/ucr/cs/riple/injector/RemovalTest.java [hunk 1]
```diff
@@ -162,7 +162,7 @@ public void removeMethodInlineAnnot() {
             "package edu.ucr;",
             "import edu.custom.Annot;",
             "public class Test {",
-            "   public@Annot Object test() {",
+            "   public @Annot Object test() {",
```
Comment (msridhar) [inline injector/src/test/java/edu/ucr/cs/riple/injector/RemovalTest.java:165]: Is this change because we changed how we insert annotations to have a leading space?

File: injector/src/test/java/edu/ucr/cs/riple/injector/RemovalTest.java [hunk 2]
```diff
@@ -187,7 +187,7 @@ public void removeParamAnnotEndingWithUnderlineExists() {
             "import javax.annotation.Nullable;",
             "import com.customAnnot.AnnotationWithNonAlphanumericSuffix$$$$_;",
             "public class Test {",
-            "   public void foo(@AnnotationWithNonAlphanumericSuffix$$$$_@Nullable Object o) { }",
+            "   public void foo(@AnnotationWithNonAlphanumericSuffix$$$$_ @Nullable Object o) { }",
```
Comment (msridhar) [inline injector/src/test/java/edu/ucr/cs/riple/injector/RemovalTest.java:190]: same as previous question

File: injector/src/test/resources/simple.java [hunk 3]
```diff
@@ -1,5 +1,4 @@
 package injector;
-
```
Comment (msridhar) [inline injector/src/test/resources/simple.java:2]: why this change?

File: injector/src/test/resources/simple_expected_1.java [hunk 4]
```diff
@@ -1,12 +1,10 @@
 package injector;
 import javax.annotation.Nullable;
-
```
Comment (msridhar) [inline injector/src/test/resources/simple_expected_1.java:3]: why this change?

**Review by msridhar — APPROVED** [review]
Looks great to me!  I love a change that deletes so much code :-)  Just a couple of minor questions but I'm also ok with landing

File: injector/src/test/java/edu/ucr/cs/riple/injector/RemovalTest.java (hunk 1 above)
Comment (nimakarimipour) [reply injector/src/test/java/edu/ucr/cs/riple/injector/RemovalTest.java:165]: Yes exactly. With new implementation we reduced scope of annotations Injector can remove since we only remove annotations the injector added itself. And the `@Annot` in this code is not how Injector adds an annotation.

File: injector/src/test/java/edu/ucr/cs/riple/injector/RemovalTest.java (hunk 2 above)
Comment (nimakarimipour) [reply injector/src/test/java/edu/ucr/cs/riple/injector/RemovalTest.java:190]: Yes exactly.

File: injector/src/test/resources/simple.java (hunk 3 above)
Comment (nimakarimipour) [reply injector/src/test/resources/simple.java]: Left over from another change, thank you for noticing this. We used to add an extra new line for each import and this is fixed in previous PR. Brought it back. e48eb1a283e8df35293e92bc19f7e19d1b5fc038

File: injector/src/test/resources/simple_expected_1.java (hunk 4 above)
Comment (nimakarimipour) [reply injector/src/test/resources/simple_expected_1.java]: Same as above.

File: injector/src/test/java/edu/ucr/cs/riple/injector/AnnotationWithArgumentTest.java [hunk 5]
```diff
@@ -51,8 +51,7 @@ public void onFieldSuppressWarningTest() {
         .expectOutput(
             "package com.edu;",
             "public class Super {",
-            "   @SuppressWarnings(\"NullAway.Init\")",
-            "   Object h = new Object();",
+            "   @SuppressWarnings(\"NullAway.Init\") Object h = new Object();",
```
Comment (lazaroclapp) [inline injector/src/test/java/edu/ucr/cs/riple/injector/AnnotationWithArgumentTest.java:54]: Mmm... I wonder if GJF will re-insert the new line here or not. 

Feels like we'd hope it does, since it matches how developers insert these suppressions better. Not sure if anyone minds these being in the same line, though, so I'd say this is fine for now. In the future, we might have to revert this decision and do `\n` in some cases and ` ` in others (should be one character either way so it shouldn't be too complicated to track, right?)

File: injector/src/test/java/edu/ucr/cs/riple/injector/ClassSearchTest.java [hunk 6]
```diff
@@ -56,8 +56,7 @@ public void annotation_declaration() {
             "package com.test;",
             "import javax.annotation.Nullable;",
             "public @interface Main{",
-            "   @Nullable",
-            "   public String foo();",
+            "   @Nullable public String foo();",
```
Comment (lazaroclapp) [inline injector/src/test/java/edu/ucr/cs/riple/injector/ClassSearchTest.java:59]: Same here, most developers would expect this to be on a line above the method declaration.

File: injector/src/test/java/edu/ucr/cs/riple/injector/ComprehensiveTest.java [hunk 7]
```diff
@@ -68,27 +68,21 @@ public void addFieldMethodParamTest() {
             "import javax.annotation.Nullable;",
             "public class Test {",
             "   Field foo1;",
-            "   @Nullable",
-            "   Field foo2;",
+            "   @Nullable Field foo2;",
```
Comment (lazaroclapp) [inline injector/src/test/java/edu/ucr/cs/riple/injector/ComprehensiveTest.java:71]: OTOH, fields are probably better annotated in the same line unless they already have a large collection of annotations...

**Review by lazaroclapp — APPROVED** [review]
Should be fine to land, I think. My only question here is if this affects the style of the generated code _after_ GJF runs. 

I think there are some considerations long-term where we might care were the `@Nullable` annotation shows up (e.g. on it's own line above methods, but in the same line at the start for fields, unless they already have a series of annotations in which case we want to follow existing style...). Meanwhile suppressions always go in their own line. This can be something we re-consider after seeing a few internal enrollment diffs, though, because we might not care enough to make our life harder here. Also, it _might_ already be handled by GJF, but I don't think it is in all cases, since I see fields with both inline and above-line annotations in our codebase.

File: injector/src/test/java/edu/ucr/cs/riple/injector/AnnotationWithArgumentTest.java (hunk 5 above)
Comment (nimakarimipour) [reply injector/src/test/java/edu/ucr/cs/riple/injector/AnnotationWithArgumentTest.java:54]: Yes exactly, we can change this in future if this becomes an issue.
